com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -95,6 +95,10 @@ type.type = string
 type.help = which type of physics to use, 2D (default) or 3D
 type.default = 2D
 
+update_frequency.type = integer
+update_frequency.help = Number of physics updates per second. 0 means one update per game frame, using the dt passed in.
+update_frequency.default = 0
```

Comment:
Note that it's disabled by default, so we need to update our templates to set this to e.g. `60`